recipes/piawka/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set name = "piawka" %}
+{% set version = "0.7.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  path: ../
+  url: https://github.com/novikovalab/piawka/archive/refs/tags/0.7.10.tar.gz

```

```suggestion
  url: https://github.com/novikovalab/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
```
Easier to maintain by using those variables here.

---

true!